Core/Src/can_handler.c

```diff
@@ -0,0 +1,158 @@
+/**
+ * @file can_handler.c
+ * @author Hamza Iqbal
+ * @brief Source file for CAN handler.  
+ * @version 0.1
+ * @date 2023-09-22
+ * 
+ * @copyright Copyright (c) 2023
+ * 
+ */
+
+#include <stdlib.h>
+#include "can.h"
+#include "can_config.h"
+
+/* Initializes the CAN handler */
+void can_handler_init()
+{
+    // Assign memory to the queues
+    can1_incoming = malloc(sizeof(msg_queue));
```

Comment:
Why do we need to allocate heap for can queues?
1. They should probably be an RTOS thing. FreeRTOS has a dedicated Queue implementation for stuff specifically for this purpose of intertask comms
2. Regardless these should be statically allocated variables that point to heap allocated structs for memory safety